site/app/templates/forum/displayThreadList.twig

```diff
@@ -1,5 +1,5 @@
 {% for thread in thread_content %}
-    <a href="{{ thread.link }}">
+    <a href="{{ thread.link }}" onclick="loadThread(event, this, {{ thread.thread_id }});">
```

Comment:
Can we have try to avoid inline js here?

Reply:
Definitely, something that can be avoided. I'll change how that works!

Reply:
Done!